src/controllers/connection/__tests__/index.test.ts

```diff
@@ -16,7 +16,7 @@ describe('ConnectionController', () => {
       let { dbMock, mockLogger, templateMock } = withConnectionMocks()
       const controller = new ConnectionController(dbMock, templateMock, mockLogger)
       const result = await controller.listConnections().then(toHTMLString)
-      expect(result).to.equal('list_foo-verified_list')
+      expect(result).to.equal('list_"foo"-verified_list')
```

Comment:
why is the foo in  "" now? (I just can't find where it came from)

Reply:
This is from `JSON.stringify()` did not want to overdue with logic as this is mainly to increase assertation. A good attached a good example
<img width="1336" alt="Screenshot 2024-07-03 at 15 10 19" src="https://github.com/digicatapult/veritable-ui/assets/11794402/c5a12b94-af7e-4658-9619-e0a303bcd04e">